plugins/woocommerce/includes/tracks/class-wc-tracks.php

```diff
@@ -38,6 +38,7 @@ public static function get_blog_details( $user_id ) {
 				'url'            => home_url(),
 				'blog_lang'      => get_user_locale( $user_id ),
 				'blog_id'        => class_exists( 'Jetpack_Options' ) ? Jetpack_Options::get_option( 'id' ) : null,
+				'store_id'       => get_option( 'woocommerce_store_id', null ),
 				'products_count' => self::get_products_count(),
 				'wc_version'     => WC()->version,
 			);
```

Comment:
Just a note, on the line below, is that these `$blog_details` are being cached in a transient for a day.
Which means that the `store_id` won't be added until a day after they update.
This isn't a huge deal in my opinion, but drove me slightly crazy while trying to test if it was set correctly after update.

Reply:
Yes, this was confusing to me, as well! Watch out for this when testing @deltaWhiskey...

Reply:
Does the 24 hour day delay matter for this new option? If so, you could probably bust the cache if the id is not set yet?